### ISSUE TITLE
Need to set correct Content Type when deploying to Azure blob storage

### ISSUE THREAD

**Issue by xumx** [mkloubert/vs-deploy#48]
currently all files defaults to `application/octet-stream` which is not very usable.

**Comment by mkloubert** [comment]
@xumx 

Oh, that's right!

**I have released Version 7.2.0 that detects the MIME type automatically now.**

You also can set an explicit `contentType` instead in your target, if detection makes problems, e.g.:

```json
{
    "deploy": {
        // ...

        "targets": [
            {
                "type": "azureblob",
                "name": "My Azure blob storage",

                "contentType": "image/png"
            }
        ]

        // ...
    }
}
```

**Comment by xumx** [comment]
Oh wow! you are fast. Thanks
